id-repository/id-repository-core/src/main/java/io/mosip/idrepository/core/manager/CredentialStatusManager.java

```diff
@@ -137,8 +137,12 @@ public void handleNewOrUpdatedRequests() {
 	public void deleteDummyPartner(CredentialRequestStatus credentialRequestStatus) {
 		Optional<CredentialRequestStatus> idWithDummyPartnerOptional = statusRepo.findByIndividualIdHashAndPartnerId(
 				credentialRequestStatus.getIndividualIdHash(), dummyPartner.getDummyOLVPartnerId());
+		mosipLogger.debug("Entry present : "
+				+ (idWithDummyPartnerOptional.isPresent() ? idWithDummyPartnerOptional.get().getStatus()
+						: "Not found"));
 		if (idWithDummyPartnerOptional.isPresent() && !idWithDummyPartnerOptional.get().getStatus()
 				.contentEquals(CredentialRequestStatusLifecycle.FAILED.toString())) {
+			mosipLogger.debug("Inside dummy partner deletion");
```

Comment:
Don't we need the user details in the logs like it is followed here ?

https://github.com/mosip/id-repository/blob/030c536208becdb1adbe2c319ee2ee98173869ef/id-repository/id-repository-core/src/main/java/io/mosip/idrepository/core/manager/CredentialStatusManager.java#L192

Reply:
No @vishwa-vyom , it is invoked from a scheduled job always.